kmyth_log/include/kmyth_log.h

```diff
@@ -194,7 +90,7 @@
  * the log file destination will not be accessible and the user will see
  * the log entries on the console.
  */
-#define KMYTH_APPLOG_OUTPUT_MODE_DEFAULT 1
+#define APPLOG_OUTPUT_MODE_DEFAULT 1
```

Comment:
Given that this is the kmyth_log library, why did you take KMYTH out of the name here?

Reply:
I think that, as part of putting the logging routines into a separate library, I wanted to make it more generic. As we kept kmyth in the name of the library, though, this was probably an inconsistent goal.

Therefore, I have changed the following variables back to include KMYTH in the name:
    KMYTH_APPLOG_OUTPUT_MODE_DEFAULT
    KMYTH_APPLOG_SEVERITY_THRESHOLD_DEFAULT

I also changed DEFAULT_APP_NAME to "kmyth" and DEFAULT_APPLOG_PATH to "/var/log/<DEFAULT_APP_NAME>.log" (this used to be default_app and /var/log/default_app.log).